R/parameters.R

```diff
@@ -49,28 +49,35 @@
 #' * `render_diagnostics`: FALSE
 #'
 #' Far UVC Intervention Parameters:
-#' * `far_uvc_workplace`: boolean switch set to TRUE if far UVC intervention parameterised in the workplace setting using `set_ucv()`; default = FALSE
+#' * `far_uvc_joint`: boolean switch set to TRUE if far UVC intervention parameterised jointly using `set_uvc()`; default = FALSE
+#' * `far_uvc_joint_coverage`: Proportion of coverage of far UVC (must be a numeric value between 0 and 1)
```

Comment:
I think actually better as is, only because it's meant to directly match the setting-type specific ones. Going to close for now, but happy to discuss further at a later date.